hummingbot/connector/exchange/gate_io/gate_io_websocket.py

```diff
@@ -34,7 +34,9 @@ def __init__(self,
 
     @property
     def last_recv_time(self) -> float:
-        return self._ws_assistant.last_recv_time
+        if self._ws_assistant is not None:
+            self._last_recv_time = self._ws_assistant.last_recv_time
+        return self._last_recv_time
```

Comment:
It seems like the local variable `_last_recv_time` is only used for this. I think it would be better if we just remove it and return 0 when the _ws_assistant is not yet initialized.

Reply:
I didn't go for that approach because it will change the current behaviour, i.e. it will cause the exchange to use the short poll interval for the polling loop, but given that we're likely reconnecting when this happens, I think that's fine